src/common/function/src/scalars/aggregate.rs

```diff
@@ -76,31 +77,42 @@ pub(crate) struct AggregateFunctions;
 
 impl AggregateFunctions {
     pub fn register(registry: &FunctionRegistry) {
-        macro_rules! register_aggr_func {
-            ($name :expr, $arg_count :expr, $creator :ty) => {
-                registry.register_aggregate_function(Arc::new(AggregateFunctionMeta::new(
-                    $name,
-                    $arg_count,
-                    Arc::new(|| Arc::new(<$creator>::default())),
-                )));
-            };
-        }
-
-        register_aggr_func!("diff", 1, DiffAccumulatorCreator);
-        register_aggr_func!("mean", 1, MeanAccumulatorCreator);
-        register_aggr_func!("polyval", 2, PolyvalAccumulatorCreator);
-        register_aggr_func!("argmax", 1, ArgmaxAccumulatorCreator);
-        register_aggr_func!("argmin", 1, ArgminAccumulatorCreator);
-        register_aggr_func!("scipystatsnormcdf", 2, ScipyStatsNormCdfAccumulatorCreator);
-        register_aggr_func!("scipystatsnormpdf", 2, ScipyStatsNormPdfAccumulatorCreator);
-        register_aggr_func!("vec_sum", 1, VectorSumCreator);
-        register_aggr_func!("vec_product", 1, VectorProductCreator);
+        registry.register_aggregate_function(Arc::new(AggregateFunctionMeta::new(
+            "diff",
```

Comment:
I believe diff, polyval, scipystatsnormcdf and scipystatsnormpdf can all be removed, too. There are numpy functions, expected to be used in our already removed "python" feature.

Reply:
agree